### ISSUE TITLE
Open HTML Files

### ISSUE THREAD

**Issue by EaterComputer** [crc-32/BrowseNX#11]
Is it possible to open an HTML file on the sd card to run in the browser?

**Comment by crc-32** [comment]
Yes, but it wouldnt be true SD opening as none of the browsers on the switch can access the SD card

**Comment by vgmoose** [comment]
Can we host an HTTP server in the background?
